pyOCD/transport/cmsis_dap.py

```diff
@@ -65,6 +65,12 @@
                  32: CSW_SIZE32
                  }
 
+DAP_MODE_SWD = 1
+DAP_MODE_JTAG = 2
+
+STICKORUN = 0x00000002
```

Comment:
Ah sorry, should have added a snippet. If those 3 are a set of errors for one register, if we can group them via prefix CTRL_ . we got something similar for CSW_ in the lines below. A comment above would be also helpful. I had to open the reference manual to find those values and their meanings